From: Peter Chen <peter.chen@nxp.com>
Date: Mon, 1 Aug 2016 15:02:36 +0800
Subject: ethernet: marvell: mvneta: add missing of_node_put after calling
 of_parse_phandle
Patch-mainline: v4.8-rc1
Git-commit: d4e4da00b68079dc13b15565a52d81f88bb61b2a
References: fate#319899

of_node_put needs to be called when the device node which is got
from of_parse_phandle has finished using.

Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Mian Yousaf Kaukab <yousaf.kaukab@suse.com>
---
 drivers/net/ethernet/marvell/mvneta.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/net/ethernet/marvell/mvneta.c
+++ b/drivers/net/ethernet/marvell/mvneta.c
@@ -4118,6 +4118,7 @@ static int mvneta_probe(struct platform_
 			pp->bm_priv = NULL;
 		}
 	}
+	of_node_put(bm_node);
 
 	err = mvneta_init(&pdev->dev, pp);
 	if (err < 0)
